index.js

```diff
@@ -1,9 +1,10 @@
 /* eslint-env node */
 'use strict';
+const whenEnabled = require('@cardstack/plugin-utils/when-enabled');
 
-module.exports = {
+module.exports = whenEnabled({
```

Comment:
This is how plugins can cause their ember-side code to only be included when the plugin itself is enabled.